openstack/examples/README.md

```diff
@@ -11,71 +11,44 @@ Guide for the Impatient
 The Ansible modules in this playbook require the `shade` Python module.
 
 Before running the playbook you must [setup your OpenStack environment variables](http://docs.openstack.org/user-guide/common/cli_set_environment_variables_using_openstack_rc.html).
-You will need to edit some of the variables in the file to match your OpenStack installation: `omero_vm_key_name`, `omero_vm_image`, `omero_vm_flavour`.
-Then run:
-
-    ansible-playbook idr-os-playbook.yml
+You can override variables at the command line, for example (note double quoting is necessary if spaces are present):
 
-Alternatively you can override variables at the command line, for example (note double quoting is necessary if spaces are present):
-
-    ansible-playbook idr-os-playbook.yml -e "omero_vm_key_name='My Keypair'"
+    ansible-playbook os-create.yml -e omero_vm_name=FOO -e omero_vm_key_name=YOURKEY \
+        -e "omero_vm_flavour='m2.xxlarge'"
 
 If this step fails it could be due to an incorrect variable, the Ansible `os_server` module usually gives an uninformative "Error in creating instance" message.
 If the VM was created the floating IP of the VM will be printed out.
 
-The `idr-example-omero-bootstrap.sh` script will continue to run in the background on the VM.
-To monitor it's progress see the VM logs in the OpenStack Horizon web interface, or on the command line (there isn't a follow option):
-
-    watch nova console-log --length 10 test-idr-omero
-
-To see the full logs:
-
-    nova console-log test-idr-omero
-
-Alternatively you can `ssh` in and tail the cloud-init logs or the system logs:
-
-    ssh centos@IP_ADDRESS tail -f /var/log/cloud-init-output.log
-    ssh centos@IP_ADDRESS sudo journalctl -f
-
-The advantage of running this playbook from inside the VM instead of pushing it out is that no SSH authentication needs to be setup, and it is not necessary to wait for the VM to initialise its SSH server.
-
 To delete the VM and related security group:
 
-    ansible-playbook idr-os-playbook-delete.yml
+    ansible-playbook os-delete.yml
 
+Inventory
+---------
 
-Updating an OpenStack VM
-------------------------
-
-You can take advantage of an [Ansible dynamic inventory script for OpenStack (`openstack.py`)](http://docs.ansible.com/ansible/intro_dynamic_inventory.html#example-openstack-external-inventory-script) instead of having to manage an inventory file when using Ansible to push out changes.
+This directory takes advantage of an
+[Ansible dynamic inventory script for OpenStack (`openstack.py`)](http://docs.ansible.com/ansible/intro_dynamic_inventory.html#example-openstack-external-inventory-script)
+instead of having to manage an inventory file when using Ansible to push out changes.
 For example:
 
-    ANSIBLE_ROLES_PATH=../../ansible/roles ansible-playbook -i openstack.py -l test-idr-omero idr-example-omero.yml -u centos -vv
+    ANSIBLE_ROLES_PATH=../../ansible/roles ansible-playbook -i inventory -l centos os-omero.yml -vv
```

Comment:
Could probably use some discussion on that. The reason that it's `-l centos` is because that chooses `ansible_ssh_user` in `inventory/variables`.